mscore/shortcut.cpp

```diff
@@ -3644,6 +3644,17 @@ void Shortcut::setKeys(const QList<QKeySequence>& ks)
       }
 
 //---------------------------------------------------------
+//   setStandardKey
+//---------------------------------------------------------
+
+void Shortcut::setStandardKey(QKeySequence::StandardKey k)
+      {
+      _standardKey = k;
+      if (_action && k != QKeySequence::UnknownKey)
+            _action->setShortcut(_standardKey);
```

Comment:
As mentioned on IRC, it should be `setShortcuts()` here, as with `setShortcut()` only one of possibly multiple shortcuts is set. And only one is shown in Preferences/Shortcuts, e.g. on Windows for File Close only Strg+F4, not also the other Ctrl+W (I guess for Mac it is the other way round). Maybe also another `for` loop is needed in keysToString()?